Wrappers/Python/cil/optimisation/algorithms/SIRT.py

```diff
@@ -77,21 +77,22 @@ class SIRT(Algorithm):
 
     The preconditioning arrays (weights) :code:`M` and :code:`D` used in SIRT are defined as
 
-    .. math:: M = \frac{1}{A*\mathbb{1}} = \frac{1}{\sum_{j}a_{i,j}}
+    .. math:: M = \frac{1}{A\mathbb{1}} = \frac{1}{\sum_{j}a_{i,j}}
 
-    .. math:: D = \frac{1}{A*\mathbb{1}} = \frac{1}{\sum_{i}a_{i,j}}
+    .. math:: D = \frac{1}{A^T\mathbb{1}} = \frac{1}{\sum_{i}a_{i,j}}
 
 
     Examples
     --------
-    .. math:: \underset{x}{\mathrm{argmin}} \frac{1}{2}\| x - d\|^{2}
+    .. math:: \underset{x}{\mathrm{argmin}} \frac{1}{2}\| Ax - d\|^{2}
 
     >>> sirt = SIRT(initial = ig.allocate(0), operator = A, data = d, max_iteration = 5)
```

Comment:
```suggestion
    >>> sirt = SIRT(initial = ig.allocate(0), operator = A, data = d)
```

I think we're moving away from using max_iteration?

Reply:
Great spot!